### DISCUSSION TITLE
Where to install / run ebusd?

### DISCUSSION THREAD

**Discussion by BlackEvidence** [john30/ebusd#892] [Q&A]
I'm fine with flashing firmware, etc. of the adapter it self. But for me it's not clear where to install / run the ebusd - after more than one day reading on here...

It's possible to run it direct on the ebus-adapter (with https://github.com/john30/ebusd-esp/)? Or is there absolutely necessary one more device like a another ESP32?

Simple goal should be, to get MQTT-data in HA....

Cheers

**Comment by Koky05** [comment]
You can run ebusd as docker container on same HW as you run HA. I do.not know what type of HA do you use. For example I run one container with HA and second with ebusd. If you use HA supervised you can find ebusd in HACS as Addon.

**Comment by BlackEvidence** [comment]
That would be too good to be true... I can't find any addon like ebus or ebusd in the addon store of HACS?!?!

And yes, I operate HA supervised on a RasPi.

**Reply by Koky05** [reply]
Here is link:
`https://github.com/LukasGrebe/ha-addons`

It was in issue #889

**Comment by BlackEvidence** [comment]
Damn! Found it in the meantime... :-/

Thx! I'll take a look.
